apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,38 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I should be able to undo action after clicking remove button', async () => {
```

Comment:
Test fails with `Failed: element not interactable` :(

I actually think that this is due to the fact that you're not waiting for the confirmation before showing the snackbar (see comments in the files below)